### PR TITLE
Fixes missing post titles in the P2P metabox

### PR DESCRIPTION
**Pull request by clontztopher**
- Based on [external PR](https://github.com/moderntribe/harvard-school-public-health/pull/23)

### CODE REVIEW

File: src/P2P/Titles_Filter.php [hunk 1]
```diff
@@ -30,6 +30,11 @@ public function __construct( array $connection_types ) {
 	 * @filter p2p_connected_title
 	 */
 	public function filter_connection_name( $title, $object, $connection_type ) {
+
+		if ( empty($title) && $object instanceof \WP_Post ) {
```
Comment (defunctl) [inline src/P2P/Titles_Filter.php:34]:
```suggestion
		if ( empty( $title ) && $object instanceof \WP_Post ) {
```

File: src/P2P/Titles_Filter.php [hunk 2]
```diff
@@ -55,6 +60,11 @@ public function filter_connection_name( $title, $object, $connection_type ) {
 	 * @filter p2p_candidate_title
 	 */
 	public function filter_candidate_name( $title, $object ) {
+
+		if ( empty($title) && $object instanceof \WP_Post ) {
```
Comment (defunctl) [inline src/P2P/Titles_Filter.php:64]:
```suggestion
		if ( empty( $title ) && $object instanceof \WP_Post ) {
```

**Review by defunctl — CHANGES_REQUESTED** [review]
Just some minor formatting but this looks good. Thanks!

**Review by dpellenwood — APPROVED** [review]
Once you've made the formatting changes suggested, you should be good to merge. Thanks Chris!

**Comment by defunctl** [comment]
I'll do a release along with my queues branch once this is in there.

**Pull request merged**